### DISCUSSION TITLE
Search Code API: Exclude repositories from org?

### DISCUSSION THREAD

**Discussion by sdruskat** [community/community#21903] [New to GitHub]
<p>Dear community,</p>
<p>I’m running a query against the API v3, such as the following:</p>
<pre><code>https://api.github.com/search/code?q=some-phrase+extension:myext
</code></pre>
<p>Works fine of course, but I’d like to exclude repositories from a specific organization. Is that possible?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>No, there isn’t a way to do that in the search API. You’ll have to look through the results and specifically exclude the ones from the organization you don’t want.</p>
